numpyro/primitives.py

```diff
@@ -66,6 +66,15 @@ def sample(name, fn, obs=None, sample_shape=()):
     additional side effects when wrapped inside effect handlers like
     :class:`~numpyro.handlers.substitute`.
 
+    .. note::
+        By design, `sample` primitive is meant to be used inside a NumPyro model.
+        Then `seed` handler is used to inject a random state to `fn`. Because
+        there is no global random state in NumPyro, NumPyro distributions'
+        `__call__` method requires an explicit keywork `random_state`. Hence, to
```

Comment:
keywork -> keyword

> use `sample` outside of NumPyro model, we can use the following pattern

I think its best to not advertise this convoluted pattern and instead recommend users to just call the distribution's `sample` method with the rng key.